src/NexusMods.Paths/FileSystemAbstraction/IFileSystem.cs

```diff
@@ -222,6 +222,15 @@ public interface IFileSystem
     /// <returns></returns>
     Task WriteAllTextAsync(AbsolutePath path, string text, CancellationToken cancellationToken = default);
 
+    /// <summary>
+    /// Writes all text to a file using UTF-8 encoding.
+    /// </summary>
+    /// <param name="path">Path to the file.</param>
+    /// <param name="text">Text to write to the file.</param>
+    /// <param name="cancellationToken">Optional <see cref="CancellationToken"/>.</param>
```

Comment:
Should be removed.

Reply:
Why?

Reply:
Without some amount of sync files we cause deadlocks in the some small situations where we need to not bounce to another thread while generating a file. Removing this and making `EnsureFiles` use `.Wait()` in "StubbedGame" causes a deadlock on systems (like CI) where we have 2 cores.

Reply:
This was regarding the `cacnellationToken` parameter, which is unused for the sync methods. GitHub doesn't like it when someone just selects 1 line and shows multiple, so the context for this was likely weird.

Reply:
Ah, makes sense 😆